### ISSUE TITLE
i have some problems in how to evaluate segmentation accuracy in eval.py

### ISSUE THREAD

**Issue by MHX1203** [milesial/Pytorch-UNet#91]
### Original code in eval.py
`for true_mask in true_masks:
                mask_pred = (mask_pred > 0.5).float()
                if net.n_classes > 1:
                    tot += F.cross_entropy(mask_pred.unsqueeze(dim=0), true_mask.unsqueeze(dim=0)).item()
                else:
                    tot += dice_coeff(mask_pred, true_mask.squeeze(dim=1)).item()`

### the above code why not like this
`for true_mask,  pred in zip(true_masks, mask_pred):
                pred = (pred > 0.5).float()
                if net.n_classes > 1:
                    tot += F.cross_entropy(pred .unsqueeze(dim=0), true_mask.unsqueeze(dim=0)).item()
                else:
                    tot += dice_coeff(pred , true_mask.squeeze(dim=1)).item()`

**Comment by milesial** [comment]
Wow you are right! Sorry, I refactored the code recently and I missed that it seems ! Also, most will train with a batch of 1 so it wouldn't have a problem for them haha.
Fixed it in d4b8040
